packages/core-data/src/selectors.js

```diff
@@ -56,12 +56,21 @@ export const isRequestingEmbedPreview = createRegistrySelector(
  * @return {Array} Authors list.
  */
 export function getAuthors( state ) {
-	deprecated( "select( 'core' ).getAuthors()", {
-		alternative: "select( 'core' ).getUsers({ who: 'authors' })",
-	} );
 	return getUserQueryResults( state, 'authors' );
 }
 
+/**
+ * Returns all available authors.
+ *
+ * @param {Object} state Data state.
+ *
+ * @return {Array} Authors list.
+ */
+export function __unstableGetAuthor( state ) {
+	const authors = getUserQueryResults( state, 'author' );
+	return authors ? authors[ 0 ] : [];
```

Comment:
It should probably return `null`  if the author was not found.

Reply:
Good point, will change.